index.html

```diff
@@ -32,34 +34,188 @@ <h2>I am a Software Developer</h2>
+                        industry's standard</p>
+                    <ul class="programming-languages">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="button" type="button">See Project</button>
+                </article>
+                <article class="projects">
+                    <h2>Professional Art <br> Printing Data</h2>
+                    <p>A daily selection of privately personalized needs; no
+                        accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul class="programming-languages">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="button" type="button">See Project</button>
+                </article>
+                <article class="projects">
+                    <h2>Professional Art <br> Printing Data</h2>
+                    <p>A daily selection of privately personalized needs; no
+                        accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul class="programming-languages">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="button" type="button">See Project</button>
+                </article>
+                <article class="projects">
+                    <h2>Professional Art <br> Printing Data</h2>
+                    <p>A daily selection of privately personalized needs; no
+                        accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul class="programming-languages">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="button" type="button">See Project</button>
+                </article>
+                <article class="projects">
+                    <h2>Professional Art <br> Printing Data</h2>
+                    <p>A daily selection of privately personalized needs; no
+                        accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul class="programming-languages">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="button" type="button">See Project</button>
+                </article>
+            </section> <!-- end work section -->
+
+            <!-- start about me section -->
+            <section class="about-me">
```

Comment:
I suggest you add the background image that is present in the figma design. 
This is a requirement for this project
<img width="1433" alt="Screenshot 2022-12-02 at 14 39 40" src="https://user-images.githubusercontent.com/48546992/205298047-ab6dfe2b-e9bc-4fc1-82bb-7b69230d46f8.png">
- This is your design
<img width="332" alt="Screenshot 2022-12-02 at 15 59 23" src="https://user-images.githubusercontent.com/48546992/205298542-01f31c95-f2c9-4324-8331-41f436fc1fa3.png">


- This is the figma design
<img width="377" alt="Screenshot 2022-12-02 at 14 47 28" src="https://user-images.githubusercontent.com/48546992/205298576-64409db3-67f7-42cb-84c1-af79c4fb7c15.png">